core/bash_impl.py

```diff
@@ -30,6 +32,20 @@ def BigInt_LessEq(a, b):
     return not mops.Greater(a, b)
 
 
+class ArrayIndexEvaluator(object):
+    """Abstract base class for things runnable in another process."""
```

Comment:
This comment should be changed, it looks like copy and paste

Reply:
Ah, yes. I'll fix it.

Reply:
I fixed it in 850af58fe.